src/Components/CreateImageWizard/ValidatedInput.tsx

```diff
@@ -38,48 +47,89 @@ type HookValidatedInputPropTypes = TextInputProps &
     inputType?: 'textInput' | 'textArea';
   };
 
-export const HookPasswordValidatedInput = ({
-  ariaLabel,
-  placeholder,
-  dataTestId,
+type ValidatedPasswordInput = TextInputProps & {
+  value: string;
+  stepValidation: StepValidation;
+  fieldName: string;
+};
+
+type PasswordInputProps = TextInputProps & {
+  value: string;
+  errorMessage: string;
+};
+
+type ErrorMessageProps = {
+  errorMessage: string;
+};
+
+export const ValidatedPasswordInput = ({
   value,
-  ouiaId,
   stepValidation,
   fieldName,
   onChange,
-  warning = undefined,
-  inputType,
-  isDisabled,
-}: HookValidatedInputPropTypes) => {
+  onBlur,
+  placeholder,
+}: ValidatedPasswordInput) => {
+  const errorMessage = stepValidation.errors[fieldName];
+  const hasError = errorMessage !== '';
+
+  return (
+    <>
+      <PasswordInput
+        value={value}
+        placeholder={placeholder || ''}
```

Comment:
Why the `|| ''` here? Can we not just pass `undefined` down to the `<TextInput>`?